Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -64,7 +64,7 @@ CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = $(CLANG_INSTRUMENT_FOR_OPTIMIZATIO
 CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING_ENABLE_LLVM_PROFILE_GENERATION = YES;
 
 GCC_PREFIX_HEADER = JavaScriptCorePrefix.h;
-GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(ENABLE_LLVM_PROFILE_GENERATION);
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(ENABLE_LLVM_PROFILE_GENERATION) PAS_BMALLOC_HIDDEN;
```

Comment:
In WebKit, we have the `BUILDING_WEBKIT` macro, which is one way to make headers behave differently when building the library than when using it. I don't have a string opinion, but wanted to suggest `BUILDING_JAVASCRIPTCORE` as a potential alternative to `PAS_BMALLOC_HIDDEN`.

...actually we also have `BUILDING_WebKit` with a different capitalization...

Reply:
I wanted to avoid a layering violation—why should bmalloc headers behave in a unique way when JavaScriptCore is building?—but if you think that's better than a bespoke new configuration macro, I'm happy to do it.

Reply:
I don't have a preference, just wanted to point out an alternative to consider.